analyzers/tests/SonarAnalyzer.Test/TestCases/DeadStores.RoslynCfg.cs

```diff
@@ -1515,4 +1515,51 @@ public void TryReturns_Loop()
 
         private void SomethingThatCanThrow() { }
     }
+
+    public class PeachValidation
+    {
+        // https://github.com/SonarSource/sonar-dotnet/issues/9471
+        void Repro9471(bool condition)
+        {
+            byte[]? tempArray = null;
```

Comment:
Do we need `byte[]?` to complicate this case, or would string be enough? This is not nullable codebase anyway

Reply:
I meant `string` instead of `string[]`. To get rid of the array, and nullability